.github/commands/gemini-triage.toml

```diff
@@ -32,23 +34,45 @@ You are an issue triage assistant. Analyze the current GitHub issue and identify
+*   *Example: `kind/bug` means an error in existing code; it excludes new feature requests.*
+
+**2. Apply General Principles:**
+Next, consider these principles to guide your decision:
+*   **Precision and Relevance:** Aim for high signal. In most cases, 1-3 labels are sufficient. It is better to apply no label than an incorrect one.
+*   **Heuristics for Priority:** If priority labels exist, map them to keywords. Terms like "security," "vulnerability," "data loss," "crash," or "outage" suggest a high priority.
+*   **Distinguishing `bug` vs. `enhancement`:** If the issue describes behavior that contradicts documentation, it's likely a `bug`. If it proposes new functionality, it's an `enhancement`.
+*   **Handling Ambiguity:** If the issue is unclear or lacks information, check for a label like `status/needs-more-information` or `status/needs-triage`. Apply it and nothing else.
+*   **Assess Issue Quality:** If the issue is not an actionable task (e.g., a question, spam), do not apply any labels.
+
+## Steps & Output Format
 
-1. Review the issue title, issue body, and available labels provided above.
+Your final output **MUST** adhere to the format specified in the **Critical Constraints**.
 
-2. Based on the issue title and issue body, classify the issue and choose all appropriate labels from the list of available labels.
+**Step 1: Reason in a comment.**
+First, in a shell comment block, summarize the results of your Reasoning Process. Justify your chosen labels by **citing specific evidence or keywords** from the issue's title or body. If you choose not to apply any labels, explain why.
 
-3. Convert the list of appropriate labels into a comma-separated list (CSV). If there are no appropriate labels, use the empty string.
+**Step 2: Generate the command.**
+Immediately after your reasoning, provide the single `echo` command to set the labels. Convert your chosen labels into a comma-separated list (CSV). If you chose no labels, use an empty string.
 
-4. Use the "echo" shell command to append the CSV labels to the output file path provided above:
+### Example Output Structure
 
-    ```
-    echo "SELECTED_LABELS=[APPROPRIATE_LABELS_AS_CSV]" >> "[filepath_for_env]"
-    ```
+**Example 1: Applying Labels**
+```
+# Reasoning: The issue describes a "crash" when a user clicks the "submit" button. This indicates a high-priority bug that affects the UI. The labels "kind/bug", "priority/p1", and "area/ui" are the most appropriate.
 
-    for example:
+echo "SELECTED_LABELS=kind/bug,priority/p1,area/ui" >> "[filepath_for_env]"
+```
 
-    ```
-    echo "SELECTED_LABELS=bug,enhancement" >> "/tmp/runner/env"
-    ```
+**Example 2: Not Applying Labels**
+```
+# Reasoning: This issue is a question about project setup ("how do I install...") and does not represent an actionable bug or feature request. Therefore, no labels are being applied.
+
+echo "SELECTED_LABELS=" >> "[filepath_for_env]"
+```
 """
```

Comment:
it's unclear if this output will also execute the echo command, did this work well when you tested it?